website/dashboard/test_plots.js

```diff
@@ -6,14 +6,14 @@ document.addEventListener('DOMContentLoaded', function() {
                 var reference_values = data.map(d => d[parameter]);
                 var fit_values = data.map(d => d[parameter + '_fit']);
                 var errors = fit_values.map((d, i) => d - reference_values[i]);
-
-                var tolerance = reference_values.map((d, i) => data[i]['atol'][parameter] + data[i]['rtol'][parameter] * d);
-                var negative_tolerance = tolerance.map(t => -t);
-
                 var minRefValue = Math.min(...reference_values);
                 var maxRefValue = Math.max(...reference_values);
                 // Create a range for tolerance trace x axis.
                 var xRange = [minRefValue, maxRefValue];
+
+                var tolerance = xRange.map((d, i) => data[reference_values.indexOf(d)]['atol'][parameter] + data[i]['rtol'][parameter] * d);
```

Comment:
I think we had also talked about plotting one or multiple bounds? What do you think @oliverchampion Currently this is somewhat randomly choosing the bounds as the bounds of the values at the endpoints.

Reply:
@AhmedBasem20, do you have the link to the latest dashboard?
@etpeterson For unit testing (errors), I think having 1 bound is probably okay for now. When we go into functional testing, we could add f-dependent and SNR-dependent bounds, but I'm also unsure how one would intuitively visualize multiple bounds...

Reply:
So the question is, what bound should we choose? When fixed it will be whatever datapoint is at the beginning or end. That's somewhat random so should it be the max or min y-extent?

Reply:
> @AhmedBasem20, do you have the link to the latest dashboard?

I'm deploying it, it will be available in ~30 minutes here: https://ahmedbasem20.github.io/TF2.4_IVIM-MRI_CodeCollection/dashboard/

Reply:
That error was something that I've been meaning to fix. I think it should be fixed here, but it will take a while to see. So you can merge that into yours if it's working. https://github.com/OSIPI/TF2.4_IVIM-MRI_CodeCollection/pull/82/

Reply:
it is now working [here](https://ahmedbasem20.github.io/TF2.4_IVIM-MRI_CodeCollection/dashboard/), please let me know what you think, thanks.